feature_aggregation_scripts/ocean_subBasin_regions/ocean_subBasin_regions.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
```

Comment:
Please move this file to `geometric_features/aggregation/ocean` and rename the file `subbasin.py` (all lowercase).  Please add a file in the `ocean` subdirectory called `__init__.py` and put in it:
``` python
from geometric_features.aggregation.ocean.subbasin import subbasin
```
Please put an empty file called `__init__.py` in the `geometric_features/aggregation` subdirectory.

Please remove execute permission from this file in its new location.

These steps will make this file a module within `geometric_features` instead of a script.